scipy/stats/tests/test_distributions.py

```diff
@@ -9646,6 +9646,53 @@ def test_sf(self):
         assert_allclose(sf1, sf0)
 
 
+def test_irwinhall():
+    unif = stats.uniform(0, 1)
+    ih1 = stats.irwinhall(1)
+
+    pts = np.linspace(0, 1, 100)
+    pdf_unif = unif.pdf(pts)
+    pdf_ih1 = ih1.pdf(pts)
+    assert_allclose(pdf_ih1, pdf_unif) # should be identical to uniform
```

Comment:
```suggestion
    assert_allclose(pdf_ih1, pdf_unif)  # should be identical to uniform
```
Minor, and I guess the linter doesn't enforce it, but would be preferred per [PEP8](https://peps.python.org/pep-0008/#inline-comments).

Reply:
I only added the comment in one place, but it is applicable elsewhere.

Reply:
Hm I'm going off the linter; does it need updating?

Reply:
The rule was enforced by our old linter, but it is in a beta state in Ruff:
https://docs.astral.sh/ruff/rules/too-few-spaces-before-inline-comment/

Even when it becomes officially supported, we might not add it - there are probably a lot of exceptions in the codebase, and we don't want people to have to fix unrelated lines whenever they touch a file. But it's easy enough to enforce in new code on a best-effort basis.